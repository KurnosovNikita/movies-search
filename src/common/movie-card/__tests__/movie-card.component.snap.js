import React from 'react';
import {shallow} from 'enzyme';
import MovieCard from '../movie-card.component';

describe('MovieCard component', () => {
    let props;
    beforeEach(() => {
        props = {
            movie: {
                id: 505,
                genres: ['drama', 'comedy', 'triller'],
                title: 'Harry Potter',
                release_date: '20.05.2018',
                poster_path: 'https://puu.sh/Dl8Lx/d4fcc4e5e6.png',
            },
        };
    });

    it('should render <MovieCard> correctly', () => {
        const component = shallow((<MovieCard {...props} />));

        expect(component).toMatchSnapshot();
    });
});
