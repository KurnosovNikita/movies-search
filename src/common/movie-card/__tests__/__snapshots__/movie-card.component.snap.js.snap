// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard component should render <MovieCard> correctly 1`] = `
<div
  className="card"
  data-movie-id={505}
>
  <Link
    to="/film/505"
  >
    <Image
      className="card-image"
      src="https://puu.sh/Dl8Lx/d4fcc4e5e6.png"
    />
  </Link>
  <div
    className="card-context"
  >
    <div
      className="card-title"
    >
      Harry Potter
    </div>
    <div
      className="card-subtitle"
    >
      drama & comedy & triller
    </div>
    <span
      className="card-label"
    >
      NaN
    </span>
  </div>
</div>
`;
